react/features/lobby/components/native/LobbyScreen.js

```diff
@@ -26,18 +25,15 @@ class LobbyScreen extends AbstractLobbyScreen {
         const { _meetingName, t } = this.props;
 
         return (
-            <CustomDialog
-                onCancel = { this._onCancel }>
-                <View style = { styles.contentWrapper }>
-                    <Text style = { styles.dialogTitle }>
-                        { t(this._getScreenTitleKey()) }
-                    </Text>
-                    <Text style = { styles.secondaryText }>
-                        { _meetingName }
-                    </Text>
-                    { this._renderContent() }
-                </View>
-            </CustomDialog>
+            <View style = { styles.contentWrapper }>
```

Comment:
Ditto, why no JitsiScreen?

Reply:
We don't need the structure applied for the JitsiScreen wrapper in this case.